draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2560,7 +2584,7 @@ to P-521 is given in {{sswu-map-to-3mod4}}.
 
 This section defines ciphersuites for curve25519 and edwards25519 {{!RFC7748}}.
 Note that these ciphersuites SHOULD NOT be used when hashing to ristretto255
-{{?I-D.irtf-cfrg-ristretto255}}.
+{{I-D.irtf-cfrg-ristretto255-decaf448}}.
```

Comment:
```suggestion
{{?I-D.irtf-cfrg-ristretto255-decaf448}}.
```

Shouldn't we use the "auto" reffing if possible? (Or is that broken because this document is too new?)

Reply:
It was broken at the time. It might be working now? I'll try to build without it and remove the manual entry if so.

Reply:
It works now!